tools/star.wdl

```diff
@@ -512,7 +512,7 @@ task alignment {
         File star_db_tar_gz
         Array[File]+ read_one_fastqs_gz
         Array[String]+ read_groups
-        Array[File] read_two_fastqs_gz = []
+        Array[File]+? read_two_fastqs_gz
```

Comment:
Interesting. Do the engines interpret this correctly? I'm assuming it should read as "optional, but if present, it must not be empty".

Reply:
Yes, it does seem to work with both miniwdl and sprocket